app/assets/stylesheets/components/_space-misc.scss

```diff
@@ -1,5 +1,6 @@
 .mxn-tiny { margin-left: -$space-tiny; margin-right: -$space-tiny; }
 .mb-tiny { margin-bottom: $space-tiny; }
+.mr-tiny { margin-right: $space-tiny; }
 .mt-tiny { margin-top: $space-tiny; }
 .pb-tiny { padding-bottom: $space-tiny; }
 .pt-tiny { padding-top: $space-tiny; }
```

Comment:
I wonder if we should just build out the whole suite of `.m-tiny` and `.p-tiny` and move it to _space-addon.scss?  May be misleading that we have `.mr-tiny` and not `.ml-tiny` for example.